barretenberg_wrapper/.cargo/config.toml

```diff
@@ -0,0 +1,2 @@
+[target.x86_64-unknown-linux-gnu]
```

Comment:
I believe we'll want to add something similar to for `aarch64-unknown-linux-gnu` in PR #8